auth/src/main/java/com/firebase/ui/auth/ui/phone/PhoneVerificationActivity.java

```diff
@@ -386,7 +383,7 @@ private void showLoadingDialog(String message) {
 
     private void dismissLoadingDialog() {
         if (mProgressDialog != null) {
-            mProgressDialog.dismiss();
+            mProgressDialog.dismissAllowingStateLoss();
```

Comment:
This would have prevented literally thousands of crashes in Santa Tracker :-( :santa: :mrs_claus: